data/glsk-file/pom.xml

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>farao-data</artifactId>
+        <groupId>com.farao-community.farao</groupId>
+        <version>1.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>farao-glsk-file</artifactId>
+    <name>GLSK file</name>
+    <description>Module that regroups all features related to GLSK file</description>
+
+    <build>
```

Comment:
This part of the POM file is not needed cause redondant with the one already defined in parent POM. To be removed

Reply:
ok